### PR TITLE
fix: prevent TypeError: Cannot read property '_target' of undefined while 'maxJoints' is too small

### PR DESCRIPTION
**Pull request by colorhook**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our [guidelines](https://github.com/oasis-engine/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

修复官方「骨骼动画」demo 在有些平台抛出的 TypeError 异常

### Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)

No

### Other information:

### CODE REVIEW

File: packages/core/src/mesh/SkinnedMeshRenderer.ts [hunk 1]
```diff
@@ -136,6 +136,7 @@ export class SkinnedMeshRenderer extends MeshRenderer {
       if (joints.length > maxJoints) {
         if (rhi.canIUseMoreJoints) {
           this._useJointTexture = true;
+          this.createJointTexture();
```
Comment (zhuxudong) [inline packages/core/src/mesh/SkinnedMeshRenderer.ts:139]: 这里的确修复了 bug，但是会多执行一次 `setPixelBuffer`，可以将 `setTexture` 等操作放到 `createJointTexture` 中执行

**Review by zhuxudong — CHANGES_REQUESTED** [review]

File: packages/core/src/mesh/SkinnedMeshRenderer.ts (hunk 1 above)
Comment (colorhook) [reply packages/core/src/mesh/SkinnedMeshRenderer.ts]: 已根据建议优化并验证

**Review by GuoLei1990 — APPROVED** [review]

File: packages/core/src/mesh/SkinnedMeshRenderer.ts (hunk 1 above)
Comment (zhuxudong) [reply packages/core/src/mesh/SkinnedMeshRenderer.ts]: 还有点小问题，给您提 PR了：https://github.com/colorhook/oasis-engine/pull/1

Comment (colorhook) [reply packages/core/src/mesh/SkinnedMeshRenderer.ts]: 666

**Review by zhuxudong — APPROVED** [review]

**Pull request merged**